src/govuk/settings/warnings.test.js

```diff
@@ -0,0 +1,62 @@
+const sass = require('node-sass')
+const { renderSass } = require('../../../lib/jest-helpers')
+
+// Create a mock warn function that we can use to override the native @warn
+// function, that we can make assertions about post-render.
+const mockWarnFunction = jest.fn()
+  .mockReturnValue(sass.NULL)
+
+const sassConfig = {
+  outputStyle: 'compressed',
+  functions: {
+    '@warn': mockWarnFunction
+  }
+}
+
+describe('Warnings mixin', () => {
+  const sassBootstrap = '@import "settings/warnings";'
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('Fires a @warn with the message plus the key suffix text', async () => {
+    const sass = `
+    ${sassBootstrap}
+    @include _warning('test', 'This is a warning.');`
+
+    await renderSass({ data: sass, ...sassConfig }).then(() => {
+      // Expect our mocked @warn function to have been called once with a single
+      // argument, which should be the test message
+      return expect(mockWarnFunction.mock.calls[0][0].getValue())
+        .toEqual(
+          'This is a warning. To silence this warning, update ' +
+          '$govuk-suppressed-warnings with key: "test"'
+        )
+    })
+  })
+
+  it('Only fires one @warn per warning key', async () => {
+    const sass = `
+    ${sassBootstrap}
+    @include _warning('test', 'This is a warning.');`
```

Comment:
I think for that test to test what it announces, we need to call the mixin twice. Otherwise, we'll get one call, but because there was only one call.

```suggestion
    @include _warning('test', 'This is a warning.');`
    @include _warning('test', 'This is a warning.');`
```